src/main/resources/hibernate.properties

```diff
@@ -17,3 +17,6 @@ hibernate.c3p0.max_size=20
 hibernate.c3p0.acquire_increment=1
 hibernate.c3p0.timeout=1800
 hibernate.c3p0.max_statements=150
+
+#Liquibase properties
+spring.liquibase.enabled=true
```

Comment:
either do explain what "enabled" means in this context or use the existing terminology - should_run